drkafka/config/doctorkafka.properties

```diff
@@ -42,54 +61,125 @@ doctorkafka.brokerstats.consumer.ssl.truststore.location=truststore_path
-doctorkafka.emails.alert=email_address_3,email_address_4
 
-# [optional] brokerstats.version
-doctorkafka.brokerstats.version=0.2.4.8
+# operator configs
+## BrokerReplacer config
+### [required]
+doctorkafka.operators.broker_replacer.class=com.pinterest.doctorkafka.modules.operator.cluster.kafka.BrokerReplacer
+### [optional]
+doctorkafka.operators.broker_replacer.config.replacement.cooldown.seconds=43200
 
+## NoStatsBrokersOperator config
+### [required]
+doctorkafka.operators.no_stats_brokers_operator.class=com.pinterest.doctorkafka.modules.operator.cluster.kafka.NoStatsBrokersOperator
+
+## URPReassignor config
+### [required]
+doctorkafka.operators.urp_reassignor.class=com.pinterest.doctorkafka.modules.operator.cluster.kafka.URPReassignor
+# please override per cluster based on fleet capacity
+doctorkafka.operators.urp_reassignor.config.network.bandwidth.max.mb=200
+### [optional]
+doctorkafka.operators.urp_reassignor.config.rack.awareness.enabled=false
+doctorkafka.operators.urp_reassignor.config.prolong.urp.alert.seconds=7200
+
+
+# action module configs
+#
+# actions will listen to subscribed events and act whenever it receives an event
+#
+# actions are in dry run mode by default
+# disable dry run mode by setting dryrun=false explicitly
+
+## report_operation action
+### [required]
+doctorkafka.actions.report_operation.class=com.pinterest.doctorkafka.modules.action.KafkaReportOperation
+doctorkafka.actions.report_operation.config.subscribed.events=report_operation
+doctorkafka.actions.report_operation.config.topic=operator_report
+doctorkafka.actions.report_operation.config.zkurl=zookeeper001:2181,zookeeper002:2181,zookeeper003:2181/cluster1
+### [optional]
+doctorkafka.actions.report_operation.config.dryrun=true
+doctorkafka.actions.report_operation.config.producer.security.protocol=SSL
+doctorkafka.actions.report_operation.config.producer.ssl.client.auth=required
+doctorkafka.actions.report_operation.config.producer.ssl.enabled.protocols=TLSv1.2,TLSv1.1,TLSv1
+doctorkafka.actions.report_operation.config.producer.ssl.endpoint.identification.algorithm=HTTPS
+doctorkafka.actions.report_operation.config.producer.ssl.key.password=key_password
+doctorkafka.actions.report_operation.config.producer.ssl.keystore.location=keystore_path
+doctorkafka.actions.report_operation.config.producer.ssl.keystore.password=keystore_password
+doctorkafka.actions.report_operation.config.producer.ssl.keystore.type=JKS
+doctorkafka.actions.report_operation.config.producer.ssl.secure.random.implementation=SHA1PRNG
+doctorkafka.actions.report_operation.config.producer.ssl.truststore.location=truststore_path
+doctorkafka.actions.report_operation.config.producer.ssl.truststore.password=truststore_password
+doctorkafka.actions.report_operation.config.producer.ssl.truststore.type=JKS
+
+## replace_instance action
+### [required]
+doctorkafka.actions.replace_instance.class=com.pinterest.doctorkafka.modules.action.cluster.ScriptReplaceInstance
```

Comment:
same here.  is `com.pinterest.doctorkafka.modules.action.cluster.ScriptReplaceInstance` the default value that others can use?